cmake/modules/Libdispatch.cmake

```diff
@@ -77,6 +77,8 @@ foreach(sdk ${DISPATCH_SDKS})
                           -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}
                           -DCMAKE_SYSTEM_NAME=${SWIFT_SDK_${sdk}_NAME}
                           -DCMAKE_SYSTEM_PROCESSOR=${arch}
+                          -DCMAKE_ANDROID_NDK="${SWIFT_ANDROID_NDK_PATH}"
```

Comment:
Quotes around the full argument (from `-D`) or removed, otherwise the quotes are part of the value.